### DISCUSSION TITLE
wrong order of arguments in tutorial

### DISCUSSION THREAD

**Discussion by helloiamstefan** [Nixtla/hierarchicalforecast#80] [Ideas]
The last line of the first big block of the tutorial on the readme file is the following:

Y_rec_df = hrec.reconcile(Y_hat_df, Y_df_train, S, tags)

It seems to me that, Y_df_train is in the wrong position here, and should come after tags (since the arguments are positional and thus the order matters)

**Comment by kdgutier** [comment]
Hi @stefanhuber1510,

Thanks for the comment, we fixed the `reconcile` method's input order in the README.
If you are interested in other running examples we have some in the documentation:
https://nixtla.github.io/hierarchicalforecast/

**Reply by mergenthaler** [reply]
Hi @stefanhuber1510, you could also name the arguments like this `Y_rec_df = hrec.reconcile(Y_hat_df=Y_hat_df, Y_df_train=Y_df_train, S=S, tags=tags)`.
